pkg/machines/components/vm/bootOrderModal.jsx

```diff
@@ -0,0 +1,319 @@
+        heading = rephraseUI("bootableDisk", "disk");
+        addOptional(additionalInfo, device.device.source.file, _("File"));
+        addOptional(additionalInfo, device.device.source.dev, _("Device"));
+        addOptional(additionalInfo, device.device.source.protocol, _("Protocol"));
+        addOptional(additionalInfo, device.device.source.pool, _("Pool"));
+        addOptional(additionalInfo, device.device.source.volume, _("Volume"));
+        addOptional(additionalInfo, device.device.source.host.name, _("Host"));
+        addOptional(additionalInfo, device.device.source.host.port, _("Port"));
+        break;
+    }
+    case "network": {
+        heading = rephraseUI("bootableDisk", "network");
+        addOptional(additionalInfo, device.device.mac, _("Mac"));
+        break;
+    }
+    case "redirdev": {
+        heading = rephraseUI("bootableDisk", "redirdev");
+        addOptional(additionalInfo, device.device.type, _("Type"));
+        addOptional(additionalInfo, device.device.bus, _("Bus"));
+        addOptional(additionalInfo, device.device.address.port, _("Port"));
+        break;
+    }
+    case "hostdev": {
+        heading = rephraseUI("bootableDisk", "hostdev");
+        const nodeDev = findHostNodeDevice(device.device, nodeDevices);
+        if (nodeDev) {
+            switch (device.device.type) {
+            case "usb": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.vendor._value, _("Vendor"));
+                addOptional(additionalInfo, nodeDev.capability.product._value, _("Product"));
+                break;
+            }
+            case "pci": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.vendor._value, _("Vendor"));
+                addOptional(additionalInfo, nodeDev.capability.product._value, _("Product"));
+                break;
+            }
+            case "scsi": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, device.device.source.address.bus, _("Bus"));
+                addOptional(additionalInfo, device.device.source.address.target, _("Target"));
+                addOptional(additionalInfo, device.device.source.address.unit, _("Unit"));
+                break;
+            }
+            case "scsi_host": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, device.device.source.protocol, _("Protocol"));
+                addOptional(additionalInfo, device.device.source.wwpn, _("WWPN"));
+                break;
+            }
+            case "mdev": {
+                addOptional(additionalInfo, device.device.type, _("Type"));
+                addOptional(additionalInfo, nodeDev.capability.type.id, _("Type ID"));
+                break;
+            }
+            }
```

Comment:
Do you want to add some default here with some debugLog that it's not handled? Or you handle everything?

Reply:
According to https://libvirt.org/formatdomain.html#elementsHostDev and https://github.com/libvirt/libvirt/blob/master/docs/schemas/domaincommon.rng
these 5 types should be all there is.